Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -629,6 +629,22 @@ private bool PerformCashSync()
                     return false;
                 }
 
+                // check if the brokerage is able to perform cash sync at this time
+                try
+                {
+                    if (!_brokerage.CanPerformCashSynchronization)
+                    {
+                        Log.Trace("BrokerageTransactionHandler.PerformCashSync(): Brokerage cannot perform cash sync at this time");
+                        _syncedLiveBrokerageCashToday = true;
```

Comment:
This seems misleading. [Here](https://github.com/QuantConnect/Lean/pull/3375/files#diff-70048ea847fdd91ea15231ea598ecfd1R635) we say we can't perform the cash sync but then [here](https://github.com/QuantConnect/Lean/pull/3375/files#diff-70048ea847fdd91ea15231ea598ecfd1R638) we say we just performed it today. I'm almost thinking it's better to constantly recheck every _n_ minutes or so until we can perform the cash sync instead of giving up for an entire day.